LUA/blockControl.lua

```diff
@@ -322,7 +332,8 @@ local function run ()
   
   showSignalStatus()                    -- Show current signal status of all block signals
 
-  local fr = 0                          -- Initialization of count of free routes
+  local available = {} 					-- Stores available routes. Per EEPmain() cycle only one route will be randomly selected fom this table
```

Comment:
We do not need variable 'fr' which is identical to '#available' if we reset this table properly.